res/freedesktop/solvespace-snap.desktop

```diff
@@ -3,7 +3,7 @@ Version=1.0
 Name=SolveSpace
 Comment=A parametric 2d/3d CAD
 Exec=solvespace
-MimeType=application/x-solvespace
+MimeType=application/x-solvespace %f
```

Comment:
I don't think that mimetype need parameter, look like it need in line 5

Reply:
Gah good catch

Reply:
Fixed!